backend/app/testbot/server.py

```
@@ -3,16 +3,124 @@
 import logging
 import websockets
 from typing import Dict, Any
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)